src/validate.ts

```diff
@@ -159,25 +160,44 @@ export class CertiMintValidation {
             anchor.merkleRoot
           );
 
-          isValid = isValid && anchor.exists && validProof;
+          isValid = anchor.exists && validProof;
```

Comment:
REQ: you broke the logic here, you allow for isValid to go from false -> true if it's followed by a valid proof and/or anchor. The `isValid = isValid &&` trick basically ensures that once invalid you can never go back to valid :) it was there for a reason